src/test/extension.test.ts

```diff
@@ -1,15 +1,100 @@
+import { createStubExtensionContext, createStubSourcegraphAPI } from '@sourcegraph/extension-api-stubs'
 import expect from 'expect'
 import mock from 'mock-require'
-import { createMockSourcegraphAPI, projects } from './stubs'
 
-const sourcegraph = createMockSourcegraphAPI()
-// For modules importing Range/Location/URI/etc
+export const sourcegraph = createStubSourcegraphAPI()
+// For modules importing Range/Location/Position/URI/etc
 mock('sourcegraph', sourcegraph)
 
-import { decorateEditor, decorateLine, getDecorations } from '../extension'
+import { activate, decorateEditor, decorateLine, getDecorations } from '../extension'
+import { resolveSettings, SentryProject } from '../settings'
 
-describe('extension', () => {
-    it('works', () => void 0)
+describe('check for extension activation', () => {
+    const context = createStubExtensionContext()
```

Comment:
This should be part of the test

Reply:
Oh as in check that `createStubExtensionContext()` doesn't return an error?

Reply:
No, but there shouldn't be logic in `describe()`. `describe()` should only contain other `describe()`, `it()` and `before`/`after` blocks. So this should be either in the test or in a `before`. Since it's only used in one test it should be in the test.